src/Main.java

```diff
@@ -1,6 +1,120 @@
+        System.out.println("Значение переменной e1 равно " + e1);
+        System.out.println("Значение переменной f1 равно " + f1);
+        System.out.println("Значение переменной g1 равно " + g1);
+
+        //Задание 3:
+        byte studentsL = 23;
+        byte studentsA = 27;
+        byte studentsE = 30;
+        short sheetTotal = 480;
+        short sheetPerStudent = (short) (sheetTotal / (studentsL + studentsA + studentsE));
+        System.out.println("На каждого ученика рассчитано " + sheetPerStudent + " листов бумаги");
+
+        //Задание 4:
+        byte bottle = 16;
+        byte time = 2;
+        byte efficiencyPerMin = (byte) (bottle / time);
+        short efficiencyPer20 = (short) (efficiencyPerMin * 20);
+        short efficiencyPerDay = (short) (efficiencyPerMin * 24 * 60);
+        int efficiencyPer3Day = efficiencyPerMin * 3 * 24 * 60;
+        int efficiencyPer1Month = efficiencyPerMin * 30 * 24 * 60;
+        System.out.println("За 20 минут машина произвела " + efficiencyPer20 + " штук бутылок");
+        System.out.println("За 1 день машина произвела " + efficiencyPerDay + " штук бутылок");
+        System.out.println("За 3 дня машина произвела " + efficiencyPer3Day + " штук бутылок");
+        System.out.println("За 1 месяц машина произвела " + efficiencyPer1Month + " штук бутылок");
+
+        //Задание 5:
+        byte totalBottles = 120;
+        byte whiteBottlePerClass = 2;
+        byte brownBottlePerClass = 4;
+        int brownWhiteBottlesPerClass = whiteBottlePerClass + brownBottlePerClass;
+        int classes = totalBottles / brownWhiteBottlesPerClass;
+        int totalWhiteBottles = whiteBottlePerClass * classes;
+        int totalBrownBottles = brownBottlePerClass * classes;
+        System.out.println("В школе, где " + classes + " классов, нужно " + totalWhiteBottles + " банок белой краски и " + totalBrownBottles +" банок коричневой краски");
+
+        //Задача 6:
+        byte bananas = 5;
+        byte weightOneBanana = 80;
+        short milkMl = 200;
+        byte weightMilkPer100Ml = 105;
+        byte iceCream = 2;
+        byte weightOneIceCream = 100;
+        byte egg = 4;
+        byte weightOneEgg = 70;
+        int breakfastInGram = (bananas * weightOneBanana) + (milkMl / 2 * weightMilkPer100Ml) + (iceCream * weightOneIceCream) + (egg * weightOneEgg);
+        short gramInKilogram = 1000;
+        float breakfastInKilogram = (float) breakfastInGram / gramInKilogram;
+        System.out.println("Вес спортзавтрака в граммах = " + breakfastInGram);
+        System.out.println("Вес спортзавтрака в килограммах = " + breakfastInKilogram);
+
+        //Задача 7:
+        byte aimInKg = 7;
+        short planLossGrA = 250;
+        short planLossGrB = 500;
+        int aimInGr = aimInKg * gramInKilogram;
+        int daysPlanA = aimInGr / planLossGrA;
+        int daysPlanB = aimInGr / planLossGrB;
+        float avPlan = (float) (planLossGrB + planLossGrA) / 2;
```

Comment:
```suggestion
        float avPlan = planLossGrB + planLossGrA / 2F;
```
можно так тоже, покороче